.gitignore

```diff
@@ -24,3 +24,4 @@ bin/
 .classpath
 .project
 .settings/
+.vscode/
```

Comment:
Could you create a new section for vscode please!

Reply:
noted!